Module.h

```diff
@@ -4,9 +4,6 @@
 #include <yarp/os/RFModule.h>
 #include <yarp/sig/Vector.h>
 
-// Workaround for https://github.com/robotology/yarp/issues/1844
-#include <yarp/os/Vocab.h>
```

Comment:
To make sure that anyone using an oldish YARP gets a  clear error message, I would suggest to also increase the version of YARP requested in https://github.com/vvv-school/tutorial_impedance-control/commit/b84b2394f80454972ce2021aba571a9957ea6a1c#diff-af3b638bc2a3e6c650974192a53c7291R5 . Unfortunately I noticed that  https://github.com/robotology/yarp/issues/1844 is still not part of any released version of YARP, so there is way of checking this. Not a big problem, but it is better to remember to update  https://github.com/vvv-school/tutorial_impedance-control/commit/b84b2394f80454972ce2021aba571a9957ea6a1c#diff-af3b638bc2a3e6c650974192a53c7291R5  as soon as  YARP 3.2 or 3.1.1 gets released.